### ISSUE TITLE
publish materials on JOSE

### ISSUE THREAD

**Issue by micaela-matta** [MDAnalysis/WorkshopHackathon2018#45]
https://jose.theoj.org/about

the material would be published as "computational learning modules, created as open educational resources (OER)".

**Comment by orbeckst** [comment]
Do we need to add a bit more guidelines for instructors on how to use the materials. The guidelines say it should be a self-contained learning module that can be directly adapted by instructors.

Do you have an example of something similar in JOSE?

**Comment by richardjgowers** [comment]
I think it will definitely require going through and updating the notebooks
with the feedback we got from the workshop.

I also think it would be cool if we can make (as many as possible) work
within Binder, so people can follow the workshop without an installation.

This might be tricky for the notebooks which required the larger datasets,
but maybe we can find a light trajectory they still work with?

**Comment by richardjgowers** [comment]
@orbeckst https://github.com/applied-bioinformatics/An-Introduction-To-Applied-Bioinformatics

this looks like the most similar example to our base (the journal only has 9 papers so far, so slim pickings).  But the description matches us well:

"Examples include Jupyter notebooks or plaintext/markup language documents like LaTeX, R Markdown, and ReST for course/lesson content and associated notes, with embedded or associated code snippets/programs.
...
tl;dr: your course or lesson content must contain or use code to teach. We are not focused exclusively on learning to code, but coding to learn."

Would it be ok if I hijacked the MDAnalysis/MDAnalysisTutorial repo and migrated all the MDAnalysis/pmda notebooks into there?

**Comment by orbeckst** [comment]
Yes.

And then we use https://nbsphinx.readthedocs.io/en/0.3.5/ :-).

We can expand the tutorial to have beginner, intermediate, advanced. Perhaps have subdirectories for these?

**Comment by orbeckst** [comment]
"migrate" = copy (leave the original Workshop stuff in place)
